src/utils/helpers/clampNumbers.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * caps the value on a lower and an upper bound
```

Comment:
Since you're introducing a new utility function, I'm going to be more picky about comments than I might be otherwise. :D

Please capitalize.